package.json

```diff
@@ -249,9 +249,9 @@
     "scripts": {
         "vscode:prepublish": "npm run compile",
         "clean": "rimraf ./out",
-        "compile": "npm run clean && tsc -p ./",
-        "watch": "npm run clean && tsc -watch -p ./",
-        "pretest": "npm run compile",
+        "compile": "npm run clean && webpack --mode production",
+        "watch": "npm run clean && webpack --mode development --watch",
```

Comment:
Something about this `--watch` switch seems to break the configurations in `launch.json`. It seems to just leave the build hanging and the sandboxed VS Code never starts up.

I checked out the sample at https://github.com/microsoft/vscode-extension-samples/tree/master/webpack-sample and it seems like they provide different scripts for `webpack` (development build, no watch), `webpack-dev` (development build, watch) and `test-compile` (just tsc, no webpack), and then adjust the launch configurations to match. It might be better to follow that example than try to create a mix of our existing scripts and Webpack-enable them.

Reply:
I think I got it fixed, could you please try again 🙇 

The documentation is problematic https://github.com/microsoft/vscode-docs/issues/3689